docs/general/configuration_options.md

```diff
@@ -28,6 +28,31 @@ Possible values:
 
 When `false`, serializers must be explicitly specified.
 
+##### belongs_to_uses_id_on_self
+
+Enable automatic serializer lookup when using a `belongs_to` without `has_many`.
+
+Possible values:
+
+- `true`
+- `false` (default)
+
+When `false`, serializers must be explicitly specified in order to use a custom `type` field in the serializer.
+
+```ruby
+class PostSerializer <  ActiveModel::Serializer
+  belongs_to :blog, type: 'custom-type'
+end
+```
+
+or to explicitly state the serializer:
+
+```ruby
+class PostSerializer <  ActiveModel::Serializer
+  belongs_to :foo, serializer: FooSerializer
```

Comment:
examples should both be belongs to :blog, but one uses the `type` option and one uses the `serializer` option, and then show how they differ in their serialized relationship given a Post with blog_id = 1 and different BlogSerializer configurations, and what happens when both are given, or if the relationship is included...